### DISCUSSION TITLE
How to display content from component in a module?

### DISCUSSION THREAD

**Discussion by FabMueh** [joomengine/Joomla-Component-Builder#666] [Q&A]
Hi, I just created a simple component which allows me to add items with several fields. Now I'd like to use the items I created in a module. My question: how can I access the components "database" within the module? Thank's in advance :-)

**Comment by Llewellynvdm** [comment]
The simplest path will be to use the dynamicGet to pull the data you would like from the component to a module. You will need to watch some more tutorial to fully understand how the dynamicGet's work.

There has been some conversation about this if I remember correctly. So checkout the following:
https://github.com/vdm-io/Joomla-Component-Builder/discussions/656
https://github.com/vdm-io/Joomla-Component-Builder/discussions/654

**Reply by FabMueh** [reply]
Thank's for your answer! To be honest, I haven't understood how to pull the data via dynamicGet directly from my component. Instead, I'm using the helper.php to connect to my database. It works pretty fine, but it might be more efficient to use dynamicGet. Anyways, thank's a lot!
